variants/RUMBA32_F446VE/PinNamesVar.h

```diff
@@ -0,0 +1,25 @@
+    /* SYS_WKUP */
```

Comment:
You could update it to add USB pin name:
https://github.com/stm32duino/Arduino_Tools/blob/master/src/genpinmap/Arduino/STM32F446V(C-E)Tx/PinNamesVar.h

Reply:
Done for USB FS (no HS on board).